config/locales/he.yml

```diff
@@ -2,245 +2,151 @@ he:
   document:
     headings:
       attachments:
-        one: "מסמך"
-        two:
-        many:
-        other: "מסמכים"
```

Comment:
Looks like we've lost a bunch of hebrew translations here, which I don't think should be removed.